rust/cmsis-pack/src/update/download.rs

```diff
@@ -130,144 +135,191 @@ where
 {
     pub fn new(config: &'a Conf, prog: Prog) -> Result<Self, Error> {
         let client = ClientBuilder::new()
-            .use_rustls_tls()
-            .use_sys_proxy()
```

Comment:
Is the proxy support enabled by default now?

Reply:
Yes